### ISSUE TITLE
Npm run build error

### ISSUE THREAD

**Issue by rockyru** [opentok/accelerator-sample-apps-js#10]
Hi, why I am getting this error?
![image](https://user-images.githubusercontent.com/13403945/29004950-540acef2-7b03-11e7-8c62-620922f844a2.png)

**Comment by maikthomas** [comment]
Hello @rockyru 

Apologies for the delayed response.

This build script will currently only work on MacOS or Linux systems. (It contains commands like `cp` instead of the windows style `copy` and `mkdir -p` instead of `mkdir`.

We are working on creating a cross-platform script to solve this issue.

In the meantime to run on windows you could edit `package.json`, changing the `cp` commands to `copy` and changing `mkdir -p` to `mkdir`.

I haven't got a windows machine to test this on right now so there could be other errors.

Best,
Mike

**Comment by maikthomas** [comment]
Note: 
That would mean changing the command `cp`, nothing to do with the `cp` in the names of the scripts.
Example:
`    "cp-annotation": "cp node_modules/opentok-annotation/dist/opentok-annotation.js public/js/components/",`
would become:
`    "cp-annotation": "copy node_modules/opentok-annotation/dist/opentok-annotation.js public/js/components/",`

**Comment by maikthomas** [comment]
Here is the contents of the whole file with those changes.
Feel free to try this and let me know if there are any other errors.

```{
  "name": "vanilla-js-sample-app",
  "version": "1.0.0",
  "description": "OpenTok Accelerator Core Vanilla JS Sample Application",
  "main": "server.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "start": "node server.js",
    "cp-annotation": "copy node_modules/opentok-annotation/dist/opentok-annotation.js public/js/components/",
    "cp-archiving": "copy node_modules/opentok-archiving/dist/opentok-archiving.js public/js/components/",
    "cp-screen-sharing": "copy node_modules/opentok-screen-sharing/dist/opentok-screen-sharing.js public/js/components/",
    "cp-text-chat": "copy node_modules/opentok-text-chat/dist/opentok-text-chat.js public/js/components/",
    "cp-logging": "copy node_modules/opentok-solutions-logging/dist/opentok-solutions-logging.js public/js/components/",
    "cp-core": "copy node_modules/opentok-accelerator-core/browser/opentok-acc-core.js public/js/components/",
    "build": "mkdir public/js/components && npm run cp-annotation && npm run cp-archiving && npm run cp-screen-sharing && npm run cp-text-chat && npm run cp-logging && npm run cp-core"
  },
  "author": "adrice727@gmail.com",
  "license": "MIT",
  "dependencies": {
    "body-parser": "^1.16.0",
    "express": "^4.14.1",
    "opentok-accelerator-core": ">=2.0.0",
    "opentok-annotation": "*",
    "opentok-archiving": "*",
    "opentok-screen-sharing": "*",
    "opentok-solutions-css": "*",
    "opentok-text-chat": "*"
  },
  "devDependencies": {
    "eslint": "^3.19.0",
    "eslint-config-airbnb": "^14.1.0",
    "eslint-plugin-import": "^2.2.0",
    "eslint-plugin-jsx-a11y": "^4.0.0",
    "eslint-plugin-react": "^6.10.3"
  }
}
```

**Comment by ksgr** [comment]
Hi @maikthomas ,
I am still getting this error..

The syntax of the command is incorrect.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! vanilla-js-sample-app@1.0.0 build: `mkdir public/js/components && npm run cp-annotation && npm run cp-archiving && npm run cp-screen-sharing && npm run cp-text-chat && npm run cp-logging && npm run cp-core`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the vanilla-js-sample-app@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:

**Comment by maikthomas** [comment]
@ksgr okay, thanks for your comment. I've made a ticket to redo the build scripts using cross-platform tools. Hopefully we'll get that out soon.

**Comment by rockyru** [comment]
I totally forgot about this. But, thanks anyway @maikthomas!

**Comment by maikthomas** [comment]
Fixed in #12
sorry for the delay!
